SerialCommand.cpp

```diff
@@ -124,6 +124,11 @@ void SerialCommand::readSerial() {
         #endif
       }
     }
+    else if (inChar == 127 || inChar == 8) {
+        bufPos--;
```

Comment:
This creates a possible buffer-overflow when deleting more characters than there are in the buffer (i.e. `bufPos` rolls over from 0 to 255).